### PR TITLE
Extract manifests

### PR DESCRIPTION
**Pull request by kat1906**
extract static content into manifests and refer to this in templates, tests and ui tests so there is one point to get this info

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
   const { filterType } = req.params;
-  const context = await getSolutionListPageContext(filterType);
+  const context = { ...await getSolutionListPageContext(filterType), ...footerLinks };
   res.render('solutions-list/template.njk', context);
 });
 
 router.get('/solutions/:filterType/:solutionId', async (req, res) => {
   const { solutionId } = req.params;
-  const context = await getPublicSolutionById(solutionId);
+  const context = { ...await getPublicSolutionById(solutionId), ...footerLinks };
```
Comment (alibombaye) [inline app/routes.js]: This is all perfect. I'm just not sure about having to pass in `footerLinks` each time to create the context.  Can we leave the footerLinks hardcoded as they was before and revist this later?

Comment (kat1906) [reply app/routes.js]: yeah I wasn't sure about it either but I kept it because it made the tests easier but can change back

Comment (alibombaye) [reply app/routes.js]: yeah I would change it back. But other than that it looks good.

Comment (kat1906) [reply app/routes.js]: 👍 going to merge now

**Pull request merged**